Characters/patch.lua

```diff
@@ -77,40 +77,275 @@ ModUtil.Path.Wrap("CalculateDamageMultipliers",
     end
 )
 
---[[ModUtil.Path.Wrap( "AddRerolls", 
+--[[ModUtil.Path.Wrap( "AddRerolls",
 	function(baseFunc, amount, source, args )
         if type(amount) == "number" then
 		    baseFunc(amount, source, args)
         end
 	end
 )]]
 function ForceNextRoomFunc(value)
-    ModUtil.Hades.PrintStackChunks(ModUtil.ToString("Click Forced:"..value))
+    ModUtil.Hades.PrintStackChunks(ModUtil.ToString("Click Forced:" .. value))
 
     -- Stomp any rooms already assigned to doors
-    for doorId, door in pairs( OfferedExitDoors ) do
+    for doorId, door in pairs(OfferedExitDoors) do
         local room = door.Room
         if room ~= nil then
             --ForceNextEncounter = "MiniBossSpreadShot"
 
             local forcedRoomData = RoomData[value]
-            local forcedRoom = CreateRoom( forcedRoomData )
-            AssignRoomToExitDoor( door, forcedRoom )
+            local forcedRoom = CreateRoom(forcedRoomData)
+            AssignRoomToExitDoor(door, forcedRoom)
         end
     end
 end
-ModUtil.Path.Wrap( "BeginOpeningCodex", 
-		function(baseFunc)		
-            --PresentationNewSameGodIncrease()
-			if (not CanOpenCodex()) and IsSuperValid() then
-				BuildSuperMeter(CurrentRun, 50)
-			end
-            --CreateAnimation({ Name = "HeraWings", DestinationId = CurrentRun.Hero.ObjectId })            
-            --ForceNextRoomFunc("B_Shop01")
-                --ModUtil.Hades.PrintStackChunks(ModUtil.ToString.TableKeys(CurrentRun.Hero.Traits))
-            
-			--LoadMap({ Name ="E_Story01", ResetBinks = true, ResetWeaponBinks = true })
-            --LoadMap({ Name ="A_Shop01", ResetBinks = true, ResetWeaponBinks = true })
-			baseFunc()
-		end
-	)
+
+ModUtil.Path.Wrap("BeginOpeningCodex",
+    function(baseFunc)
+        --PresentationNewSameGodIncrease()
+        if (not CanOpenCodex()) and IsSuperValid() then
+            BuildSuperMeter(CurrentRun, 50)
```

Comment:
Don't forget to comment out

Reply:
Those comments are still useful for testing Makaria. I didn't add them I just moved them from Hestia code to Patch since they will affect both Hera and Hestia's code. They will be removed or moved to Makaria by the time Hera is released. Is that okay?

Reply:
alright!